base/applications/cacls/lang/af-ZA

```diff
@@ -0,0 +1,72 @@
+LANGUAGE LANG_AFRIKAANS, SUBLANG_AFRIKAANS_SOUTH_AFRICA
```

Comment:
This seems to be a duplicate of af-ZA.rc?

Reply:
There seem to be a lot of these extension-less files.

Reply:
Because #1 I put files without extension ".rc".
So, #2 I revert all files.
#3 are the final files with extension ".rc".